### PR TITLE
added visual feedback.

### LINKED ISSUE #1: Give visual feedback when a correct or wrong answer is provided

### PR DESCRIPTION
**Pull request by 0xSk3ttch**
Sorry it took so long! Ran into some stuff around the house that took precedence. If you've got any input or would like any changes, let me know.

### CODE REVIEW

**Comment by chowder** [comment]
Thanks a lot for this, Jacob. 

This looks good for the most part - though in the preview it doesn't seem like the input box's ring colour is updating as you intend it to?

**Comment by 0xSk3ttch** [comment]
I'll check again in a bit. It was a quick green for correct and a quick red for incorrect.

**Comment by 0xSk3ttch** [comment]
@chowder Will dig into this deeper. I'm not sure how Netlify deployments work. All is well locally, but those colors aren't updating on the deployed version. Classes are being applied as they should.

File: src/App.js [hunk 1]
```diff
@@ -42,15 +43,14 @@ function App() {
   const submitAnswer = (answer) => {
     answer = answer.trim()
     if (currentAnswers.includes(answer)) {
-      console.log(`${answer} has already been guessed!`)
+      setErrors([...errors, `${answer} has already been guessed!`])
```
Comment (chowder) [inline src/App.js]: Why do we store a list of all the errors, instead of storing just the latest error?

Comment (0xSk3ttch) [reply src/App.js]: No particular reason, actually. I'll take it out. Still looking for whatever is happening between local and deployed. I'm stumped. Versions are the same. The class gets applied. Got any ideas?

**Comment by 0xSk3ttch** [comment]
@chowder there we go! conditionally applying the color to the class wasn't working. damn that took too long for such a small issue.

**Comment by chowder** [comment]
Thanks Jacob! 

I'll likely make some small touches to the styles, out of personal preference. But this looks good to me. :) 

Thanks again for the contribution!

**Comment by 0xSk3ttch** [comment]
No problem! I guess I should have asked first. Never worked on anything open source before. Just personal and work stuff.

**Pull request merged**